dev/managedservicesplatform/operationdocs/index_test.go

```diff
@@ -0,0 +1,49 @@
+package operationdocs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hexops/autogold/v2"
+
+	"github.com/sourcegraph/sourcegraph/dev/managedservicesplatform/spec"
+	"github.com/sourcegraph/sourcegraph/lib/pointers"
+)
+
+func TestRenderIndexPage(t *testing.T) {
+	services := []*spec.Spec{
+		{
+			Service: spec.ServiceSpec{
+				ID:     "service-1",
+				Name:   pointers.Ptr("Service 1"),
+				Owners: []string{"team-a"},
+			},
+		},
+		{
+			Service: spec.ServiceSpec{
+				ID:     "service-2",
+				Name:   pointers.Ptr("Service 1"),
+				Owners: []string{"team-a"},
+			},
+		},
+		{
+			Service: spec.ServiceSpec{
+				ID:     "service-3",
+				Name:   pointers.Ptr("Service 1"),
+				Owners: []string{"team-b"},
+			},
+		},
+		{
+			Service: spec.ServiceSpec{
+				ID:     "service-4",
+				Name:   pointers.Ptr("Service 1"),
+				Owners: []string{"team-a", "team-b"},
+			},
+		},
```

Comment:
nit: doesn't really matter but makes the test output a bit easier to verify correctness if the names are unique
```suggestion
		{
			Service: spec.ServiceSpec{
				ID:     "service-1",
				Name:   pointers.Ptr("Service 1"),
				Owners: []string{"team-a"},
			},
		},
		{
			Service: spec.ServiceSpec{
				ID:     "service-2",
				Name:   pointers.Ptr("Service 2"),
				Owners: []string{"team-a"},
			},
		},
		{
			Service: spec.ServiceSpec{
				ID:     "service-3",
				Name:   pointers.Ptr("Service 3"),
				Owners: []string{"team-b"},
			},
		},
		{
			Service: spec.ServiceSpec{
				ID:     "service-4",
				Name:   pointers.Ptr("Service 4"),
				Owners: []string{"team-a", "team-b"},
			},
		},
```

Reply:
ah good catch, thanks!